### PR TITLE
migrate_options_shared: Update config file for keepalive setting

### PR DESCRIPTION
**Pull request by cliping**
Update config file for keepalive setting in modular daemons mode.

Signed-off-by: lcheng <lcheng@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/migration/migrate_options_shared.py [hunk 1]
```diff
@@ -825,8 +825,20 @@ def update_qemu_conf_on_local_and_remote():
 
         conf_dict = eval(params.get("qemu_conf_dict", '{}'))
         conf_dest_dict = params.get("qemu_conf_dest_dict", '{}')
+        conf_type = params.get("qemu_conf_type")
+        conf_type_dest = params.get("qemu_conf_dest_type")
+
+        if not utils_split_daemons.is_modular_daemon():
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_options_shared.py:831]: maybe this logic can be put into migration_base under provider folder to commonly shared

Comment (cliping) [reply libvirt/tests/src/migration/migrate_options_shared.py]: According to new document, updated. Thanks.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]
LGTM

**Pull request merged**